tests/unit/utils/serialize/test_yaml.py

```diff
@@ -1,6 +1,10 @@
+import os
+import tempfile
+
 import pytest
 
-from dvc.utils.serialize._yaml import YAMLError, parse_yaml
+from dvc.utils.serialize._common import EncodingError
+from dvc.utils.serialize._yaml import YAMLError, load_yaml, parse_yaml
```

Comment:
We should not be using a private API to test here. As we already doing it, I'm fine with it, but we should fix it later. 🙂

Reply:
Thanks! Wasn't aware of this convention. I've updated this test to import from `dvc.utils.serialize` instead.